src/helpers.coffee

```diff
@@ -48,11 +48,14 @@ extend = exports.extend = (object, properties) ->
 # Handy for getting a list of `children` from the nodes.
 exports.flatten = flatten = (array) ->
   flattened = []
-  for element in array
-    if '[object Array]' is Object::toString.call element
-      flattened = flattened.concat flatten element
-    else
-      flattened.push element
+  stack = [array]
+  while stack.length
+    current = stack.pop()
+    for element in current
+      if '[object Array]' is Object::toString.call element
+        stack.push(element)
+      else
+        flattened.push(element)
```

Comment:
Perhaps instead we should use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat? I don’t think we even need to feature-detect it, support goes very far back.

Reply:
Yes, we can use this if old browser/Node.js support is not required.
```coffeescript
exports.flatten = flatten = (array) ->
  array.flat(10)
```
10 - is max depth

Reply:
> Yes, we can use this if old browser/Node.js support is not required.

You can look in our GitHub Actions to see our support matrix, but basically everywhere we support can run `.flat`. You can use `.flat(Infinity)` to go as far down as necessary.

Reply:
I changed it and built.
```bash
npx cake build
npx cake build:browser
```

Reply:
CI is failing. Try `cake release`?

Reply:
Done